Trippy/Utils/CoreDataUtils.m

```diff
@@ -0,0 +1,78 @@
+//
+//  CoreDataUtils.m
+//  Trippy
+//
+//  Created by Catherine Lu on 7/26/22.
+//
+
+#import "CoreDataUtils.h"
+#import "Location.h"
+#import "LocationCollection.h"
+#import "Itinerary.h"
+#import "CoreDataHandler.h"
+@import CoreData;
+
+@implementation CoreDataUtils
+
++ (Location *)locationFromManagedObject:(NSManagedObject *)obj {
```

Comment:
I feel like there should be some basic error checking in these methods, in case you feed it an invalid `NSManagedObject` (since it's basically being used as an opaque dictionary). I'm not sure if `NSManagedObject` has any sense of "what it is", if you will, but at the least you could check to see if a unique field in each of these cases is present and return nil otherwise.